src/main/java/com/dakar/dakar/controller/ReactiveController.java

```diff
@@ -88,6 +89,17 @@
                 ok().body(journeyService.saveJourney(Mono.just(new Journey(id, "afghanistan", "afghanistan", ""))), Journey.class));
     }
 
+    /**
+     * DEMO
+     * classic delete endpoint
+     */
+    @Bean
+    RouterFunction<ServerResponse> deleteJourney() {
+        return route(RequestPredicates.DELETE("/deleteJourney/{id}"), request ->
+                ServerResponse.noContent().build(journeyService.deleteJourney(request.pathVariable("id"))));
```

Comment:
can't we responde "ok" or sonmething instead of "noContent" ?

Reply:
I did this because deletion does not return sommething. This is what I found for that.